hooks/post_gen_project.py

```diff
@@ -33,3 +39,34 @@ def remove_copying_files():
 # 1. Removes files needed for the GPLv3 license if it isn't going to be used.
 if not use_gplv3:
     remove_copying_files()
+
+
+def get_node_lts():
+    url = "https://api.github.com/repos/nodejs/node/releases"
+    res = requests.get(url)
+    version = [
+        {"tag_name": v["tag_name"], "name": v["name"]}
+        for v
+        in res.json()
+        if "LTS" in v["name"]
+    ]
+    return version[0]["tag_name"].lstrip("v")
+
+
+def get_yarn_latest():
+    url = "https://api.github.com/repos/yarnpkg/yarn/releases/latest"
+    res = requests.get(url)
+    return res.json()["tag_name"].lstrip("v")
+
+
+package_json = os.path.join(PROJECT_DIRECTORY, "package.json")
+
+with open(package_json) as f:
+    data = json.load(f)
```

Comment:
```
Traceback (most recent call last):
  File "/var/folders/lc/8svywbfn33l06bgj29w0_px80000gn/T/tmpVLMOlH.py", line 65, in <module>
    data = json.load(f)
NameError: name 'json' is not defined
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
```

Reply:
Oops, forgot an import. Fixed.